### DISCUSSION TITLE
TypeError: Failed to fetch when loading the page

### DISCUSSION THREAD

**Discussion by lznbb** [GMOD/jbrowse-components#3874] [Q&A] [answered]
![image](https://github.com/GMOD/jbrowse-components/assets/99380544/325a3420-2985-4b95-9196-4fa7ca60f186)
This error occasionally shows up. After several refreshes, everything goes well. I am sure that the config.json file is ready and the genomic files are also there. Could you give any suggestions that might cause this?

**Comment by cmdcolin** [comment]
that's quite odd. can you open the javascript console and look to see if there is a stack trace by chance?

**Reply by lznbb** [reply]
ok thanks. I will try to investigate next time I run into this problem. Just for your reference, our jbrowse server is deployed in the cloud service. I suspect it could be a bit correlated with some performance metrics. Any clue that the network or payload might cause this?

**Reply by cmdcolin** [reply, marked as answer]
Somestimes "cross origin" (CORS) requests cause this. Chrome will block the browser from accessing the true error message or any info

if you are accessing any cross origin stuff, I can try to help debug that further. The network logs in your developer console e.g. devtools will likely have more info even when you get an opaque error message like this

If you are not accessing any cross origin stuff and it is still giving this error let me know

**Reply by lznbb** [reply]
Thanks for the inference. Till now the issue does not reproduce frequently. Maybe it's ok to close this ticket for now. I will let you know when I run into the problem again and see whether I can provide more console logs for debugging.
